roles/uperf/tasks/init.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Clear start flag 
+  command: "redis-cli set start 0"
+
+- name: Clear num_completion
+  command: "redis-cli set num_completion 0"
+
+- name: Init node and pod indices in benchmark context
+  operator_sdk.util.k8s_status:
+     api_version: ripsaw.cloudbulldozer.io/v1alpha1
+     kind: Benchmark
+     name: "{{ meta.name }}"
+     namespace: "{{ operator_namespace }}"
+     status:
+        pod_hi_idx: "{{pod_hi_idx}}"
+        pod_low_idx: "{{pod_low_idx}}"
+        node_hi_idx: "{{node_hi_idx}}"
+        node_low_idx: "{{node_low_idx}}"
+        node_idx: "{{node_low_idx}}"
+        pod_idx: "{{pod_low_idx}}"
```

Comment:
Why are we setting these variables in the bechmark status?

We should use redis to store these kind of data.

Reply:
There are a couple of reasons:
1. redis should be used for shared data between the operator and the workload pods. These variables are the operator's private data like state i.e "Building" - The workload pods do not use/see them.
2. redis-server has already shown performance pressure in scale runs. We must scrub all unnecessary redis usages.
3. Putting these variables in benchmark status allows a *natural* way to monitor the progress by invoking "oc describe benchmark" and see.
Reason (1) and (2) are compelling.